theano/tensor/signal/pool.py

```diff
@@ -614,6 +613,9 @@ def c_headers(self):
         headers += super(Pool, self).c_headers()
         return headers
 
+    def c_code_cache_version(self):
+        return (COp.c_code_cache_version(self), self.openmp)
```

Comment:
Checked, all OpenMPOp sub-classes take into account `self.openmp`, and for the moment I have not modified the other CPU Pool* ops.